Sources/Core/ISO8601.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+// Modified from http://stackoverflow.com/a/28016692/239380
+
+public struct ISO8601 {
+    public static let shared = ISO8601()
+    public let formatter: DateFormatter
+
+    public init() {
+        formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .iso8601)
+        formatter.locale = Locale(identifier: "en_US_POSIX")
```

Comment:
Locale should either be inferred by the system or injected in I would say. Generally for Server-side you don't want any Locale and should just stick to UTC. For giving to a client it should probably be the default from where they are visiting from